chapters/limits.txt

```diff
@@ -3370,6 +3370,28 @@ include::{generated}/validity/structs/VkPhysicalDevicePipelineRobustnessProperti
 --
 endif::VK_EXT_pipeline_robustness[]
 
+ifdef::VK_EXT_map_memory_placed[]
+[open,refpage='VkPhysicalDeviceMapMemoryPlacedPropertiesEXT',desc='Structure describing the alignment requirements of placed memory maps for a physical device',type='structs']
+--
+The sname:VkPhysicalDeviceMapMemoryPlacedPropertiesEXT structure is
+defined as:
+
+include::{generated}/api/structs/VkPhysicalDeviceMapMemoryPlacedPropertiesEXT.txt[]
+
+The members of the sname:VkPhysicalDeviceMapMemoryPlacedPropertiesEXT
+structure describe the following:
+
+  * [[limits-minPlacedMemoryMapAlignment]]
+    pname:minPlacedMemoryMapAlignment is the minimum alignment required by
+    placed memory maps.
```

Comment:
```suggestion
    memory ranges when using placed memory mapping.
```

maybe?

It wasn't immediately obvious to me if this was a requirement on the host address or the memory range - the VU clears it up but it'd be good to not have to go digging for it.

Reply:
Yeah. Good call.  I think I've fixed it.

Reply:
Ok hang on, so is it meant to be _both_? If so, then I think you need a VU to state that the address needs to be aligned too (at least that's how I'm reading your new text?)

Reply:
Yes, it's on both.  It should just be page size but I figured a limit is better than trying to refer to the system page size and hoping that's always the exact requirement.  There's already a VU on `VkMemoryMapPlacedInfoEXT`.

Reply:
Ah I missed that it was in a different place. Thanks!